tests/components/matter/test_lock.py

```diff
@@ -72,19 +76,18 @@ async def test_lock(
         command=clusters.DoorLock.Commands.LockDoor(),
         timed_request_timeout_ms=1000,
     )
-    matter_client.send_device_command.reset_mock()
 
-    await hass.async_block_till_done()
     state = hass.states.get("lock.mock_door_lock")
     assert state
     assert state.state == LockState.LOCKING
+    matter_client.send_device_command.reset_mock()
 
-    set_node_attribute(matter_node, 1, 257, 0, 0)
```

Comment:
I think you still want to have a test for the case of `NotFullyLocked`, because there is still some expected result of such a state, correct?